source/simulator/core.cc

```diff
@@ -974,10 +989,38 @@ namespace aspect
       for (unsigned int c=0; c<dim; ++c)
         for (unsigned int d=0; d<dim; ++d)
           coupling[x.velocities[c]][x.velocities[d]] = DoFTools::always;
-      for (unsigned int d=0; d<dim; ++d)
```

Comment:
Please add some sentences to the comment above these lines about the `include_melt_transport` case